setup.py

```diff
@@ -64,11 +65,9 @@
      console_scripts=[
         '{}=spinalcordtoolbox.compat.launcher:main'.format(x) for x in \
         [
-            'isct_check_detection',
             'isct_convert_binary_to_trilinear',
             'isct_minc2volume-viewer',
             'isct_test_ants',
-            'isct_warpmovie_generator',
```

Comment:
Is this because of #2828? I see the "Internal Commands" documentation section was changed, and so if I understand, this was updated to match.  Just a nitpick, but could this change to `setup.py` instead be part of a larger PR addressing #2828? Or, does the presence of these two lines affect how the documentation is built?

Reply:
> Is this because of #2828? 

yes

> I see the "Internal Commands" documentation section was changed, and so if I understand, this was updated to match. Just a nitpick, but could this change to `setup.py` instead be part of a larger PR addressing #2828? Or, does the presence of these two lines affect how the documentation is built?

it would ideally be part of a larger PR, but as you suspected, the presence of these two lines affect how the doc is built: it display an ugly message under these functions in the doc:
~~~
isct_warpmovie_generator
Traceback (most recent call last):
  File "/Users/julien/code/sct/scripts/isct_warpmovie_generator.py", line 18, in <module>
    from scipy.misc import toimage
ImportError: cannot import name 'toimage'
~~~

Reply:
Thank you for the clarification -- that makes sense. :slightly_smiling_face: